scala-package/mxnet-demo/scala-demo/Makefile

```diff
@@ -17,7 +17,6 @@
 
 SCALA_VERSION_PROFILE := 2.11
 SCALA_VERSION := 2.11.8
-MXNET_VERSION := 1.3.0
```

Comment:
Anything preventing us from also making this change in the java-demo? It'd be good to keep them as in sync as possible.

Reply:
Yep, referring to my reply above, I'll make both the "demos" in sync with each other and follow the same pattern.

Reply:
+1 for make them Sync. And due to the advantage of static build, user would likely to have less problems with SNAPSHOT or future versions